src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp

```diff
@@ -243,7 +260,28 @@ int micrortps_client_main(int argc, char *argv[])
 			PX4_INFO("Not running");
 
 		} else {
-			PX4_INFO("Running");
+			px4_clock_gettime(CLOCK_REALTIME, &end);
+
+			const double elapsed_secs = static_cast<double>(end.tv_sec - begin.tv_sec + (end.tv_nsec - begin.tv_nsec) / 1e9);
+
+			printf("\tup and running for %.03f seconds\n", elapsed_secs);
+			printf("\tnr. of messages received: %" PRIu64 "\n", received);
+			printf("\tnr. of messages sent: %" PRIu64 "\n", sent);
+			printf("\ttotal data read: %" PRIu64 " bytes\n", total_rcvd);
+			printf("\ttotal data sent: %" PRIu64 " bytes\n", total_sent);
+			printf("\trates:\n");
+			printf("\t  rx: %.3f kB/s\n", rcvd_last_sec / 1e3);
+			printf("\t  tx: %.3f kB/s\n", sent_last_sec / 1e3);
+			printf("\t  avg rx: %.3f kB/s\n", total_rcvd / (1e3 * elapsed_secs));
+			printf("\t  avg tx: %.3f kB/s\n", total_sent / (1e3 * elapsed_secs));
+			printf("\t  tx rate max:");
```

Comment:
I think you could use `PX4_INFO_RAW` instead of `printf` here, not sure if it matters.

Reply:
maybe. Let me check

Reply:
I am keeping it as it is to match what we have for the MAVLink case: https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/mavlink/mavlink_main.cpp#L2821-L2828